core/templates/dev/head/pages/skill_editor/skill_editor_navbar_breadcrumb_directive.html

```diff
@@ -1,6 +1,6 @@
 <div class="nav navbar-nav oppia-navbar-breadcrumb ng-cloak">
   <span class="oppia-navbar-breadcrumb-separator"></span>
   <span>
-    Skill Editor
+    Skill Editor > <[getTruncatedDescription()]>
```

Comment:
Just a thought that came from looking at this -- what do you think of having a separate "short description" field for skills? Most skills that we currently write are very precisely described, e.g. "Given two fractions with the same denominator, find their sum." So if we're going to show a truncated title in the breadcrumb (and perhaps elsewhere), it'll just be "Given two ..." which isn't really helpful. Maybe we should have an informal name and a longer fully-descriptive name.

What do you think? Also it might be worth experimenting with making the breadcrumb font smaller (for the skill description part) or truncating a bit less; not sure. Could also be something that depends on the window width and maybe provide accommodation to see the full thing via tooltip.

What do you think? Also /cc @tjiang11 @vinitamurthi.

Reply:
For the smaller informal name suggestion, can such a name be given to every skill? I think there might be skills where such a name might be similar for multiple skills (or else it might become long again). 
I think it might be better to decrease font-size and display more of or full skill description in the navbar, since mobile editing is not supported anyway and we are limiting skill description length to 40 which could fit in the navbar for a normal display.
I am also fine with adding a tooltip but I am not sure whether a user would think of hovering on the skill description in the navbar? Also, since the skill description is anyway shown on the skill editor page itself (where it's edited), I am not sure if a tooltip is necessary.

Reply:
Yeah, that's fair. I'm happy deferring the addition of a new smaller informal name. The idea was that that name may not be unique between skills, it would just try and convey the main points of the skill.

Rest sounds good, thanks!